doc/website/index.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: splash
+permalink: /
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/mm-home-page-feature.jpg
+  actions:
+    - label: "<i class='fas fa-download'></i> Install now"
+      url: "/get-started/setup/"
+excerpt: >
+  Develop and design Hardware seamlessly with the power of ROHD.<br />
```

Comment:
Somewhere on the home page we need to say what ROHD actually is and what it stands for.  We could take the github description to start maybe: 
> The Rapid Open Hardware Development (ROHD) framework is a framework for describing and verifying hardware in the Dart programming language. ROHD enables you to build and traverse a graph of connectivity between module objects using unrestricted software.